libs/application/templates/children-residence-change/src/fields/Confirmation/index.tsx

```diff
@@ -1,27 +1,78 @@
 import React from 'react'
 import { useIntl } from 'react-intl'
-import { Box } from '@island.is/island-ui/core'
+import { Box, Text } from '@island.is/island-ui/core'
 import { confirmation, copyUrl } from '../../lib/messages'
-import { DescriptionText, CopyUrl } from '../components'
+import { CopyUrl, DescriptionText } from '../components'
+import { CRCFieldBaseProps } from '../..'
+import ContractAccordionOverview from '../components/ContractOverviewAccordion/ContractOverviewAccordion'
 
-const Confirmation = () => {
+const Confirmation = ({ application }: CRCFieldBaseProps) => {
   const { formatMessage } = useIntl()
+  const { answers, externalData } = application
+
   return (
     <>
       <Box marginTop={3}>
         <DescriptionText
           text={confirmation.general.description}
-          format={{ applicationNumber: '12345' }}
+          format={{
```

Comment:
I really like this advanced use of translation strings but I feel like it's not very readable. Even though I know the context I still needed to check the translation string so I could fully understand what is happening. Should we add a comment or would it be possible to simplify in some way?
I don't have the solution right now and I'm just wondering 🤔

Reply:
I'll add a comment for now :)